providers/cloudant/README.rst

```diff
@@ -0,0 +1,62 @@
+
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+ .. NOTE! THIS FILE IS AUTOMATICALLY GENERATED AND WILL BE OVERWRITTEN!
+
+ .. IF YOU WANT TO MODIFY TEMPLATE FOR THIS FILE, YOU SHOULD MODIFY THE TEMPLATE
+    `PROVIDER_README_TEMPLATE.rst.jinja2` IN the `dev/breeze/src/airflow_breeze/templates` DIRECTORY
+
+
+Package ``apache-airflow-providers-cloudant``
+
+Release: ``4.1.0``
+
+
+`IBM Cloudant <https://www.ibm.com/cloud/cloudant>`__
+
+
+Provider package
+----------------
+
+This is a provider package for ``cloudant`` provider. All classes for this provider package
+are in ``airflow.providers.cloudant`` python package.
+
+You can find package information and changelog for the provider
+in the `documentation <https://airflow.apache.org/docs/apache-airflow-providers-cloudant/4.1.0/>`_.
+
+Installation
+------------
+
+You can install this package on top of an existing Airflow 2 installation (see ``Requirements`` below
+for the minimum Airflow version supported) via
+``pip install apache-airflow-providers-cloudant``
+
+The package supports the following python versions: 3.10,3.11,3.12
+
+Requirements
+------------
+
+==================  =====================================
+PIP package         Version required
+==================  =====================================
+``apache-airflow``  ``>=2.9.0``
+``ibmcloudant``     ``==0.9.1; python_version >= "3.10"``
+==================  =====================================
+
```

Comment:
NIT: An idea - this README is something immediatly visible for package in PyPI -> maybe we could also add information about Python versions supported?

Reply:
That could be a neat follow up. But it will affect all providers, perhaps we update it in a separate PR?

Reply:
Actually... I am totally stupid... It's already up there :) :+1: 

>  The package supports the following python versions: 3.10,3.11,3.12